examples/esp_idf/hello/sdkconfig.defaults

```diff
@@ -0,0 +1,47 @@
+CONFIG_MBEDTLS_SSL_PROTO_DTLS=y
+CONFIG_MBEDTLS_PSK_MODES=y
+CONFIG_MBEDTLS_KEY_EXCHANGE_PSK=y
+CONFIG_MBEDTLS_TLS_CLIENT_ONLY=y
+CONFIG_MBEDTLS_SHA512_C=n
+CONFIG_LWIP_NETBUF_RECVINFO=y
+CONFIG_FREERTOS_USE_TRACE_FACILITY=y
+CONFIG_FREERTOS_USE_STATS_FORMATTING_FUNCTIONS=y
+CONFIG_ESP_WIFI_SOFTAP_SUPPORT=n
+
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
+
+# OTA Partition Table
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_PARTITION_TABLE_CUSTOM=y
```

Comment:
Along with my comment on partitions.csv, I think we can remove this config setting.